tests/lib/scan/scan_test.py

```diff
@@ -0,0 +1,168 @@
+import unittest
+from unittest import mock
+
+from prowler.lib.scan.scan import Scan, get_service_checks_to_execute
+from tests.lib.outputs.fixtures.fixtures import generate_finding_output
+from tests.providers.aws.utils import set_mocked_aws_provider
+
+
+class Test_Scan(unittest.TestCase):
```

Comment:
Remember that we not use `unittest` we use `pytest`.

Reply:
True, test have been refactored